### ISSUE TITLE
Sort by name in /tag route returns in descending order

### ISSUE THREAD

**Issue by Tlazypanda** [publiclab/plots2#7758]
### Please describe the problem (or idea)
Usually when we sort by name we expect it to be sorted in the ascending order but in /tag route it returns in the descending order. This behaviour can be modified to return in ascending order by default instead of clicking it again to toggle the asc/desc.
![image](https://user-images.githubusercontent.com/33183263/77931773-441e0600-72ca-11ea-9e1e-008668b37000.png)

### Please show us where to look

https://publiclab.org/tags

Code in tag_controller, index method

**Comment by Tlazypanda** [comment]
@VladimirMikulic @jywarren can you kindly approve? Thanks :v: Also one more thing since there is no icon like the following next to the sort operation it isn't clear that it toggles between the ascending and descending a little design change might help users be aware of this functionality
![image](https://user-images.githubusercontent.com/33183263/77932140-c27aa800-72ca-11ea-952d-fb564b57413c.png)

**Comment by VladimirMikulic** [comment]
Very true. Thank you for addressing this issue.

**Comment by sssash18** [comment]
@VladimirMikulic @Tlazypanda Can I work on this ?

**Comment by VladimirMikulic** [comment]
@sssash18 if @Tlazypanda is not already working on this, feel free to take this use.

**Comment by Tlazypanda** [comment]
@sssash18 Hey I just started working on this issue if you don't mind can I continue working? I really need to make a good contribution for the Outreachy application :sweat_smile: If you want you can take this up #7756  Would love if you can help me with it ... currently figuring out how to tweak the toggle behaviour to have 2 default values ..would love your input. Thanks :v:

**Comment by Tlazypanda** [comment]
@VladimirMikulic  I meant this code here for the backend - in `tag_controller.rb` file :sweat_smile: 
```
def index
    @toggle = params[:sort] || "uses"

    @title = I18n.t('tag_controller.tags')
    @paginated = true
    @order_type = params[:order] == "desc" ? "asc" : "desc"
```
Here the order_type is set by default to desc and the sort is by default set to "uses" so I am not sure how to change it for the sort=="name" case.

**Comment by Tlazypanda** [comment]
@jywarren @Uzay-G can you help me out with this? :sweat_smile: Thanks :v:

**Comment by akp2603** [comment]
@cesswairimu if this issue is still open, please let me know, would like to help.
cc: @Tlazypanda @renugasaraswathy

**Comment by cesswairimu** [comment]
Yes the issue is still open @akp2603. We would love your help. Thanks

**Comment by akp2603** [comment]
@cesswairimu would love to love with the backend part of it. Can you assign me the issue?

**Comment by cesswairimu** [comment]
Great, go ahead.

**Comment by akp2603** [comment]
Hi @cesswairimu, it seems like I dont have the permission to raise a pull request against the repository. Can you please check?

**Comment by cesswairimu** [comment]
Hey @akp2603 that is strange, could you please send the link of the branch you want to make a branch from

**Comment by akp2603** [comment]
Hi @cesswairimu agreed. But I am not even able to push my local branch. Let alone raise a pull request.
```
ERROR: Permission to publiclab/plots2.git denied to akp2603.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```
This is the message I am getting. Guess you have to give me the push access.

**Comment by Cadreia** [comment]
Hi, are u sure you're online? Check your internet connection

**Comment by akp2603** [comment]
Hi @Cadreia , yes pulling the code from master works. Pushing to other repos works too. The issue is with this repo only.

**Comment by Cadreia** [comment]
@akp2603 it looks like you are trying to push to the main repository instead of your forked version. Did you fork the publiclab/plots2 repo before making changes to the code?

**Comment by akp2603** [comment]
@Cadreia Ahh thanks, I had been pushing directly to other repos. Your comment helped. Raised the PR now. Thanks for the help again. 👍

**Comment by akp2603** [comment]
Hi @cesswairimu the PR has been pending approval and merging. Sorry to bother, but the hacktoberfest deadline is quite close now. Can you please get it reviewed? Thanks :)

**Comment by cesswairimu** [comment]
Hey @akp2603, sure, I see @jywarren left some comments I will ping Jeff to do a review again. Thanks

**Comment by cesswairimu** [comment]
@akp2603, looks like Jeff may not be available I will go ahead review and merge the PR so that your contribution can be counted and I will keep this open just incase Jeff needs some follow up to  be done on this. I will ping him next week. Thanks for the patience.

**Comment by jywarren** [comment]
Noting there may be some follow-up on defaults being discussed here! https://github.com/publiclab/plots2/issues/8941

Thanks, all!

**Comment by cesswairimu** [comment]
This was resolved in https://github.com/publiclab/plots2/issues/8941. Thanks all
